[2020-06-20T00:33:28,973][WARN ][logstash.config.source.multilocal] Ignoring the 'pipelines.yml' file because modules or command line options are specified
[2020-06-20T00:33:29,085][INFO ][logstash.runner          ] Starting Logstash {"logstash.version"=>"7.7.1"}
[2020-06-20T00:33:30,624][INFO ][org.reflections.Reflections] Reflections took 49 ms to scan 1 urls, producing 21 keys and 41 values 
[2020-06-20T00:33:30,980][WARN ][logstash.outputs.elasticsearch] You are using a deprecated config setting "document_type" set in elasticsearch. Deprecated settings will continue to work, but are scheduled for removal from logstash in the future. Document types are being deprecated in Elasticsearch 6.0, and removed entirely in 7.0. You should avoid this feature If you have any questions about this, please visit the #logstash channel on freenode irc. {:name=>"document_type", :plugin=><LogStash::Outputs::ElasticSearch index=>"user_info_index", id=>"2d2a90d5f8aa8b9d46930cded2d613577bb3fc1d5a8ba8b12facb40a6a4290eb", document_id=>"%{id}", hosts=>[http://localhost:9200], document_type=>"user_info", enable_metric=>true, codec=><LogStash::Codecs::Plain id=>"plain_cc34190c-e898-4aa0-8feb-5c9d3fec45d3", enable_metric=>true, charset=>"UTF-8">, workers=>1, manage_template=>true, template_name=>"logstash", template_overwrite=>false, doc_as_upsert=>false, script_type=>"inline", script_lang=>"painless", script_var_name=>"event", scripted_upsert=>false, retry_initial_interval=>2, retry_max_interval=>64, retry_on_conflict=>1, ilm_enabled=>"auto", ilm_rollover_alias=>"logstash", ilm_pattern=>"{now/d}-000001", ilm_policy=>"logstash-policy", action=>"index", ssl_certificate_verification=>true, sniffing=>false, sniffing_delay=>5, timeout=>60, pool_max=>1000, pool_max_per_route=>100, resurrect_delay=>5, validate_after_inactivity=>10000, http_compression=>false>}
[2020-06-20T00:33:32,635][INFO ][logstash.outputs.elasticsearch][main] Elasticsearch pool URLs updated {:changes=>{:removed=>[], :added=>[http://localhost:9200/]}}
[2020-06-20T00:33:32,854][WARN ][logstash.outputs.elasticsearch][main] Restored connection to ES instance {:url=>"http://localhost:9200/"}
[2020-06-20T00:33:32,916][INFO ][logstash.outputs.elasticsearch][main] ES Output version determined {:es_version=>6}
[2020-06-20T00:33:32,920][WARN ][logstash.outputs.elasticsearch][main] Detected a 6.x and above cluster: the `type` event field won't be used to determine the document _type {:es_version=>6}
[2020-06-20T00:33:32,940][WARN ][logstash.outputs.elasticsearch][main] DEPRECATION WARNING: Connecting to an OSS distribution of Elasticsearch using the default distribution of Logstash will stop working in Logstash 8.0.0. Please upgrade to the default distribution of Elasticsearch, or use the OSS distribution of Logstash {:url=>"http://localhost:9200/"}
[2020-06-20T00:33:32,957][INFO ][logstash.outputs.elasticsearch][main] New Elasticsearch output {:class=>"LogStash::Outputs::ElasticSearch", :hosts=>["http://localhost:9200"]}
[2020-06-20T00:33:33,017][INFO ][logstash.outputs.elasticsearch][main] Using default mapping template
[2020-06-20T00:33:33,050][INFO ][logstash.outputs.elasticsearch][main] Index Lifecycle Management is set to 'auto', but will be disabled - Your Elasticsearch cluster is before 7.0.0, which is the minimum version required to automatically run Index Lifecycle Management
[2020-06-20T00:33:33,053][INFO ][logstash.outputs.elasticsearch][main] Attempting to install template {:manage_template=>{"template"=>"logstash-*", "version"=>60001, "settings"=>{"index.refresh_interval"=>"5s"}, "mappings"=>{"_default_"=>{"dynamic_templates"=>[{"message_field"=>{"path_match"=>"message", "match_mapping_type"=>"string", "mapping"=>{"type"=>"text", "norms"=>false}}}, {"string_fields"=>{"match"=>"*", "match_mapping_type"=>"string", "mapping"=>{"type"=>"text", "norms"=>false, "fields"=>{"keyword"=>{"type"=>"keyword", "ignore_above"=>256}}}}}], "properties"=>{"@timestamp"=>{"type"=>"date"}, "@version"=>{"type"=>"keyword"}, "geoip"=>{"dynamic"=>true, "properties"=>{"ip"=>{"type"=>"ip"}, "location"=>{"type"=>"geo_point"}, "latitude"=>{"type"=>"half_float"}, "longitude"=>{"type"=>"half_float"}}}}}}}}
[2020-06-20T00:33:33,062][WARN ][org.logstash.instrument.metrics.gauge.LazyDelegatingGauge][main] A gauge metric of an unknown type (org.jruby.specialized.RubyArrayOneObject) has been created for key: cluster_uuids. This may result in invalid serialization.  It is recommended to log an issue to the responsible developer/development team.
[2020-06-20T00:33:33,070][INFO ][logstash.javapipeline    ][main] Starting pipeline {:pipeline_id=>"main", "pipeline.workers"=>8, "pipeline.batch.size"=>125, "pipeline.batch.delay"=>50, "pipeline.max_inflight"=>1000, "pipeline.sources"=>["C:/Users/bahma/eclipse-workspace/project/logstash/bin/logstash-mysql.conf"], :thread=>"#<Thread:0x322f4e36 run>"}
[2020-06-20T00:33:34,240][INFO ][logstash.javapipeline    ][main] Pipeline started {"pipeline.id"=>"main"}
[2020-06-20T00:33:34,339][INFO ][logstash.agent           ] Pipelines running {:count=>1, :running_pipelines=>[:main], :non_running_pipelines=>[]}
[2020-06-20T00:33:34,762][INFO ][logstash.agent           ] Successfully started Logstash API endpoint {:port=>9600}
[2020-06-20T00:34:04,882][ERROR][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] Unable to connect to database. Tried 1 times {:error_message=>"Java::ComMysqlCjJdbcExceptions::CommunicationsException: Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server."}
[2020-06-20T00:35:04,275][ERROR][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] Unable to connect to database. Tried 1 times {:error_message=>"Java::ComMysqlCjJdbcExceptions::CommunicationsException: Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server."}
[2020-06-20T00:36:04,205][ERROR][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] Unable to connect to database. Tried 1 times {:error_message=>"Java::ComMysqlCjJdbcExceptions::CommunicationsException: Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server."}
[2020-06-20T00:37:04,127][ERROR][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] Unable to connect to database. Tried 1 times {:error_message=>"Java::ComMysqlCjJdbcExceptions::CommunicationsException: Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server."}
[2020-06-20T00:38:04,040][ERROR][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] Unable to connect to database. Tried 1 times {:error_message=>"Java::ComMysqlCjJdbcExceptions::CommunicationsException: Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server."}
[2020-06-20T00:39:04,256][ERROR][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] Unable to connect to database. Tried 1 times {:error_message=>"Java::ComMysqlCjJdbcExceptions::CommunicationsException: Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server."}
[2020-06-20T00:40:04,188][ERROR][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] Unable to connect to database. Tried 1 times {:error_message=>"Java::ComMysqlCjJdbcExceptions::CommunicationsException: Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server."}
[2020-06-20T00:41:04,096][ERROR][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] Unable to connect to database. Tried 1 times {:error_message=>"Java::ComMysqlCjJdbcExceptions::CommunicationsException: Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server."}
[2020-06-20T00:42:04,313][ERROR][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] Unable to connect to database. Tried 1 times {:error_message=>"Java::ComMysqlCjJdbcExceptions::CommunicationsException: Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server."}
[2020-06-20T00:43:04,240][ERROR][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] Unable to connect to database. Tried 1 times {:error_message=>"Java::ComMysqlCjJdbcExceptions::CommunicationsException: Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server."}
[2020-06-20T00:44:04,168][ERROR][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] Unable to connect to database. Tried 1 times {:error_message=>"Java::ComMysqlCjJdbcExceptions::CommunicationsException: Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server."}
[2020-06-20T00:45:04,085][ERROR][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] Unable to connect to database. Tried 1 times {:error_message=>"Java::ComMysqlCjJdbcExceptions::CommunicationsException: Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server."}
[2020-06-20T00:46:04,287][ERROR][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] Unable to connect to database. Tried 1 times {:error_message=>"Java::ComMysqlCjJdbcExceptions::CommunicationsException: Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server."}
[2020-06-20T00:47:04,248][ERROR][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] Unable to connect to database. Tried 1 times {:error_message=>"Java::ComMysqlCjJdbcExceptions::CommunicationsException: Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server."}
[2020-06-20T00:48:04,157][ERROR][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] Unable to connect to database. Tried 1 times {:error_message=>"Java::ComMysqlCjJdbcExceptions::CommunicationsException: Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server."}
[2020-06-20T00:49:04,066][ERROR][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] Unable to connect to database. Tried 1 times {:error_message=>"Java::ComMysqlCjJdbcExceptions::CommunicationsException: Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server."}
[2020-06-20T00:50:04,267][ERROR][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] Unable to connect to database. Tried 1 times {:error_message=>"Java::ComMysqlCjJdbcExceptions::CommunicationsException: Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server."}
[2020-06-20T00:51:04,153][ERROR][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] Unable to connect to database. Tried 1 times {:error_message=>"Java::ComMysqlCjJdbcExceptions::CommunicationsException: Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server."}
[2020-06-20T00:52:04,065][ERROR][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] Unable to connect to database. Tried 1 times {:error_message=>"Java::ComMysqlCjJdbcExceptions::CommunicationsException: Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server."}
[2020-06-20T00:53:04,268][ERROR][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] Unable to connect to database. Tried 1 times {:error_message=>"Java::ComMysqlCjJdbcExceptions::CommunicationsException: Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server."}
[2020-06-20T00:54:04,173][ERROR][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] Unable to connect to database. Tried 1 times {:error_message=>"Java::ComMysqlCjJdbcExceptions::CommunicationsException: Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server."}
[2020-06-20T00:55:04,063][ERROR][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] Unable to connect to database. Tried 1 times {:error_message=>"Java::ComMysqlCjJdbcExceptions::CommunicationsException: Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server."}
[2020-06-20T00:56:04,277][ERROR][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] Unable to connect to database. Tried 1 times {:error_message=>"Java::ComMysqlCjJdbcExceptions::CommunicationsException: Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server."}
[2020-06-20T00:57:04,176][ERROR][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] Unable to connect to database. Tried 1 times {:error_message=>"Java::ComMysqlCjJdbcExceptions::CommunicationsException: Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server."}
[2020-06-20T00:58:04,082][ERROR][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] Unable to connect to database. Tried 1 times {:error_message=>"Java::ComMysqlCjJdbcExceptions::CommunicationsException: Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server."}
[2020-06-20T00:59:04,290][ERROR][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] Unable to connect to database. Tried 1 times {:error_message=>"Java::ComMysqlCjJdbcExceptions::CommunicationsException: Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server."}
[2020-06-20T01:00:04,185][ERROR][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] Unable to connect to database. Tried 1 times {:error_message=>"Java::ComMysqlCjJdbcExceptions::CommunicationsException: Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server."}
[2020-06-20T01:01:04,105][ERROR][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] Unable to connect to database. Tried 1 times {:error_message=>"Java::ComMysqlCjJdbcExceptions::CommunicationsException: Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server."}
[2020-06-20T01:02:04,321][ERROR][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] Unable to connect to database. Tried 1 times {:error_message=>"Java::ComMysqlCjJdbcExceptions::CommunicationsException: Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server."}
[2020-06-20T01:03:04,210][ERROR][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] Unable to connect to database. Tried 1 times {:error_message=>"Java::ComMysqlCjJdbcExceptions::CommunicationsException: Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server."}
[2020-06-20T01:04:04,091][ERROR][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] Unable to connect to database. Tried 1 times {:error_message=>"Java::ComMysqlCjJdbcExceptions::CommunicationsException: Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server."}
[2020-06-20T01:05:04,269][ERROR][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] Unable to connect to database. Tried 1 times {:error_message=>"Java::ComMysqlCjJdbcExceptions::CommunicationsException: Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server."}
[2020-06-20T01:06:04,160][ERROR][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] Unable to connect to database. Tried 1 times {:error_message=>"Java::ComMysqlCjJdbcExceptions::CommunicationsException: Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server."}
[2020-06-20T01:07:04,035][ERROR][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] Unable to connect to database. Tried 1 times {:error_message=>"Java::ComMysqlCjJdbcExceptions::CommunicationsException: Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server."}
[2020-06-20T01:08:04,211][ERROR][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] Unable to connect to database. Tried 1 times {:error_message=>"Java::ComMysqlCjJdbcExceptions::CommunicationsException: Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server."}
[2020-06-20T01:09:04,099][ERROR][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] Unable to connect to database. Tried 1 times {:error_message=>"Java::ComMysqlCjJdbcExceptions::CommunicationsException: Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server."}
[2020-06-20T01:10:04,300][ERROR][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] Unable to connect to database. Tried 1 times {:error_message=>"Java::ComMysqlCjJdbcExceptions::CommunicationsException: Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server."}
[2020-06-20T01:11:04,182][ERROR][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] Unable to connect to database. Tried 1 times {:error_message=>"Java::ComMysqlCjJdbcExceptions::CommunicationsException: Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server."}
[2020-06-20T01:12:04,075][ERROR][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] Unable to connect to database. Tried 1 times {:error_message=>"Java::ComMysqlCjJdbcExceptions::CommunicationsException: Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server."}
[2020-06-20T01:13:04,268][ERROR][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] Unable to connect to database. Tried 1 times {:error_message=>"Java::ComMysqlCjJdbcExceptions::CommunicationsException: Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server."}
[2020-06-20T01:14:04,145][ERROR][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] Unable to connect to database. Tried 1 times {:error_message=>"Java::ComMysqlCjJdbcExceptions::CommunicationsException: Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server."}
[2020-06-20T01:15:04,041][ERROR][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] Unable to connect to database. Tried 1 times {:error_message=>"Java::ComMysqlCjJdbcExceptions::CommunicationsException: Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server."}
[2020-06-20T01:16:04,231][ERROR][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] Unable to connect to database. Tried 1 times {:error_message=>"Java::ComMysqlCjJdbcExceptions::CommunicationsException: Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server."}
[2020-06-20T01:17:04,092][ERROR][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] Unable to connect to database. Tried 1 times {:error_message=>"Java::ComMysqlCjJdbcExceptions::CommunicationsException: Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server."}
[2020-06-20T01:18:04,319][ERROR][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] Unable to connect to database. Tried 1 times {:error_message=>"Java::ComMysqlCjJdbcExceptions::CommunicationsException: Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server."}
[2020-06-20T01:19:04,213][ERROR][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] Unable to connect to database. Tried 1 times {:error_message=>"Java::ComMysqlCjJdbcExceptions::CommunicationsException: Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server."}
[2020-06-20T01:20:04,095][ERROR][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] Unable to connect to database. Tried 1 times {:error_message=>"Java::ComMysqlCjJdbcExceptions::CommunicationsException: Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server."}
[2020-06-20T01:21:04,292][ERROR][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] Unable to connect to database. Tried 1 times {:error_message=>"Java::ComMysqlCjJdbcExceptions::CommunicationsException: Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server."}
[2020-06-20T01:22:04,166][ERROR][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] Unable to connect to database. Tried 1 times {:error_message=>"Java::ComMysqlCjJdbcExceptions::CommunicationsException: Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server."}
[2020-06-20T01:23:04,050][ERROR][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] Unable to connect to database. Tried 1 times {:error_message=>"Java::ComMysqlCjJdbcExceptions::CommunicationsException: Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server."}
[2020-06-20T01:24:04,216][ERROR][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] Unable to connect to database. Tried 1 times {:error_message=>"Java::ComMysqlCjJdbcExceptions::CommunicationsException: Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server."}
[2020-06-20T01:25:04,093][ERROR][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] Unable to connect to database. Tried 1 times {:error_message=>"Java::ComMysqlCjJdbcExceptions::CommunicationsException: Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server."}
[2020-06-20T01:26:04,269][ERROR][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] Unable to connect to database. Tried 1 times {:error_message=>"Java::ComMysqlCjJdbcExceptions::CommunicationsException: Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server."}
[2020-06-20T01:27:04,165][ERROR][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] Unable to connect to database. Tried 1 times {:error_message=>"Java::ComMysqlCjJdbcExceptions::CommunicationsException: Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server."}
[2020-06-20T01:28:04,048][ERROR][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] Unable to connect to database. Tried 1 times {:error_message=>"Java::ComMysqlCjJdbcExceptions::CommunicationsException: Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server."}
[2020-06-20T01:29:04,217][ERROR][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] Unable to connect to database. Tried 1 times {:error_message=>"Java::ComMysqlCjJdbcExceptions::CommunicationsException: Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server."}
[2020-06-20T01:30:04,106][ERROR][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] Unable to connect to database. Tried 1 times {:error_message=>"Java::ComMysqlCjJdbcExceptions::CommunicationsException: Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server."}
[2020-06-20T01:31:04,291][ERROR][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] Unable to connect to database. Tried 1 times {:error_message=>"Java::ComMysqlCjJdbcExceptions::CommunicationsException: Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server."}
[2020-06-20T01:32:01,085][INFO ][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] (0.266604s) SELECT user_id as id,languages,middlewares,databasesList,operating_system,experience,frameworks,i.cus_name,i.cus_addr,i.cus_dob,i.cus_email,i.cus_tel FROM user_cv INNER JOIN user_info i ON user_cv.user_id = i.CUS_USER_ID WHERE user_cv.user_id > 0
[2020-06-20T01:33:00,147][INFO ][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] (0.001649s) SELECT user_id as id,languages,middlewares,databasesList,operating_system,experience,frameworks,i.cus_name,i.cus_addr,i.cus_dob,i.cus_email,i.cus_tel FROM user_cv INNER JOIN user_info i ON user_cv.user_id = i.CUS_USER_ID WHERE user_cv.user_id > 7502
[2020-06-20T01:34:00,321][INFO ][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] (0.001000s) SELECT user_id as id,languages,middlewares,databasesList,operating_system,experience,frameworks,i.cus_name,i.cus_addr,i.cus_dob,i.cus_email,i.cus_tel FROM user_cv INNER JOIN user_info i ON user_cv.user_id = i.CUS_USER_ID WHERE user_cv.user_id > 7502
[2020-06-20T01:35:00,229][INFO ][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] (0.001089s) SELECT user_id as id,languages,middlewares,databasesList,operating_system,experience,frameworks,i.cus_name,i.cus_addr,i.cus_dob,i.cus_email,i.cus_tel FROM user_cv INNER JOIN user_info i ON user_cv.user_id = i.CUS_USER_ID WHERE user_cv.user_id > 7502
[2020-06-20T01:36:00,122][INFO ][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] (0.000890s) SELECT user_id as id,languages,middlewares,databasesList,operating_system,experience,frameworks,i.cus_name,i.cus_addr,i.cus_dob,i.cus_email,i.cus_tel FROM user_cv INNER JOIN user_info i ON user_cv.user_id = i.CUS_USER_ID WHERE user_cv.user_id > 7502
[2020-06-20T01:37:00,323][INFO ][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] (0.001029s) SELECT user_id as id,languages,middlewares,databasesList,operating_system,experience,frameworks,i.cus_name,i.cus_addr,i.cus_dob,i.cus_email,i.cus_tel FROM user_cv INNER JOIN user_info i ON user_cv.user_id = i.CUS_USER_ID WHERE user_cv.user_id > 7502
[2020-06-20T01:38:00,212][INFO ][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] (0.000990s) SELECT user_id as id,languages,middlewares,databasesList,operating_system,experience,frameworks,i.cus_name,i.cus_addr,i.cus_dob,i.cus_email,i.cus_tel FROM user_cv INNER JOIN user_info i ON user_cv.user_id = i.CUS_USER_ID WHERE user_cv.user_id > 7502
[2020-06-20T01:39:00,109][INFO ][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] (0.001454s) SELECT user_id as id,languages,middlewares,databasesList,operating_system,experience,frameworks,i.cus_name,i.cus_addr,i.cus_dob,i.cus_email,i.cus_tel FROM user_cv INNER JOIN user_info i ON user_cv.user_id = i.CUS_USER_ID WHERE user_cv.user_id > 7502
[2020-06-20T01:40:00,306][INFO ][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] (0.001225s) SELECT user_id as id,languages,middlewares,databasesList,operating_system,experience,frameworks,i.cus_name,i.cus_addr,i.cus_dob,i.cus_email,i.cus_tel FROM user_cv INNER JOIN user_info i ON user_cv.user_id = i.CUS_USER_ID WHERE user_cv.user_id > 7502
[2020-06-20T01:40:30,134][WARN ][logstash.runner          ] SIGINT received. Shutting down.
[2020-06-20T01:40:35,147][WARN ][logstash.runner          ] Received shutdown signal, but pipeline is still waiting for in-flight events
to be processed. Sending another ^C will force quit Logstash, but this may cause
data loss.
[2020-06-20T01:40:35,266][WARN ][org.logstash.execution.ShutdownWatcherExt] {"inflight_count"=>0, "stalling_threads_info"=>{"other"=>[{"thread_id"=>41, "name"=>"[main]<jdbc", "current_call"=>"[...]/vendor/bundle/jruby/2.5.0/gems/rufus-scheduler-3.0.9/lib/rufus/scheduler.rb:170:in `join'"}, {"thread_id"=>33, "name"=>"[main]>worker0", "current_call"=>"[...]/logstash-core/lib/logstash/java_pipeline.rb:279:in `block in start_workers'"}, {"thread_id"=>34, "name"=>"[main]>worker1", "current_call"=>"[...]/logstash-core/lib/logstash/java_pipeline.rb:279:in `block in start_workers'"}, {"thread_id"=>35, "name"=>"[main]>worker2", "current_call"=>"[...]/logstash-core/lib/logstash/java_pipeline.rb:279:in `block in start_workers'"}, {"thread_id"=>36, "name"=>"[main]>worker3", "current_call"=>"[...]/logstash-core/lib/logstash/java_pipeline.rb:279:in `block in start_workers'"}, {"thread_id"=>37, "name"=>"[main]>worker4", "current_call"=>"[...]/logstash-core/lib/logstash/java_pipeline.rb:279:in `block in start_workers'"}, {"thread_id"=>38, "name"=>"[main]>worker5", "current_call"=>"[...]/logstash-core/lib/logstash/java_pipeline.rb:279:in `block in start_workers'"}, {"thread_id"=>39, "name"=>"[main]>worker6", "current_call"=>"[...]/logstash-core/lib/logstash/java_pipeline.rb:279:in `block in start_workers'"}, {"thread_id"=>40, "name"=>"[main]>worker7", "current_call"=>"[...]/logstash-core/lib/logstash/java_pipeline.rb:279:in `block in start_workers'"}]}}
[2020-06-20T01:40:35,273][ERROR][org.logstash.execution.ShutdownWatcherExt] The shutdown process appears to be stalled due to busy or blocked plugins. Check the logs for more information.
[2020-06-20T01:40:40,306][WARN ][org.logstash.execution.ShutdownWatcherExt] {"inflight_count"=>0, "stalling_threads_info"=>{"other"=>[{"thread_id"=>41, "name"=>"[main]<jdbc", "current_call"=>"[...]/vendor/bundle/jruby/2.5.0/gems/rufus-scheduler-3.0.9/lib/rufus/scheduler.rb:170:in `join'"}, {"thread_id"=>33, "name"=>"[main]>worker0", "current_call"=>"[...]/logstash-core/lib/logstash/java_pipeline.rb:279:in `block in start_workers'"}, {"thread_id"=>34, "name"=>"[main]>worker1", "current_call"=>"[...]/logstash-core/lib/logstash/java_pipeline.rb:279:in `block in start_workers'"}, {"thread_id"=>35, "name"=>"[main]>worker2", "current_call"=>"[...]/logstash-core/lib/logstash/java_pipeline.rb:279:in `block in start_workers'"}, {"thread_id"=>36, "name"=>"[main]>worker3", "current_call"=>"[...]/logstash-core/lib/logstash/java_pipeline.rb:279:in `block in start_workers'"}, {"thread_id"=>37, "name"=>"[main]>worker4", "current_call"=>"[...]/logstash-core/lib/logstash/java_pipeline.rb:279:in `block in start_workers'"}, {"thread_id"=>38, "name"=>"[main]>worker5", "current_call"=>"[...]/logstash-core/lib/logstash/java_pipeline.rb:279:in `block in start_workers'"}, {"thread_id"=>39, "name"=>"[main]>worker6", "current_call"=>"[...]/logstash-core/lib/logstash/java_pipeline.rb:279:in `block in start_workers'"}, {"thread_id"=>40, "name"=>"[main]>worker7", "current_call"=>"[...]/logstash-core/lib/logstash/java_pipeline.rb:279:in `block in start_workers'"}]}}
[2020-06-20T01:40:45,337][WARN ][org.logstash.execution.ShutdownWatcherExt] {"inflight_count"=>0, "stalling_threads_info"=>{"other"=>[{"thread_id"=>41, "name"=>"[main]<jdbc", "current_call"=>"[...]/vendor/bundle/jruby/2.5.0/gems/rufus-scheduler-3.0.9/lib/rufus/scheduler.rb:170:in `join'"}, {"thread_id"=>33, "name"=>"[main]>worker0", "current_call"=>"[...]/logstash-core/lib/logstash/java_pipeline.rb:279:in `block in start_workers'"}, {"thread_id"=>34, "name"=>"[main]>worker1", "current_call"=>"[...]/logstash-core/lib/logstash/java_pipeline.rb:279:in `block in start_workers'"}, {"thread_id"=>35, "name"=>"[main]>worker2", "current_call"=>"[...]/logstash-core/lib/logstash/java_pipeline.rb:279:in `block in start_workers'"}, {"thread_id"=>36, "name"=>"[main]>worker3", "current_call"=>"[...]/logstash-core/lib/logstash/java_pipeline.rb:279:in `block in start_workers'"}, {"thread_id"=>37, "name"=>"[main]>worker4", "current_call"=>"[...]/logstash-core/lib/logstash/java_pipeline.rb:279:in `block in start_workers'"}, {"thread_id"=>38, "name"=>"[main]>worker5", "current_call"=>"[...]/logstash-core/lib/logstash/java_pipeline.rb:279:in `block in start_workers'"}, {"thread_id"=>39, "name"=>"[main]>worker6", "current_call"=>"[...]/logstash-core/lib/logstash/java_pipeline.rb:279:in `block in start_workers'"}, {"thread_id"=>40, "name"=>"[main]>worker7", "current_call"=>"[...]/logstash-core/lib/logstash/java_pipeline.rb:279:in `block in start_workers'"}]}}
[2020-06-20T01:40:50,366][WARN ][org.logstash.execution.ShutdownWatcherExt] {"inflight_count"=>0, "stalling_threads_info"=>{"other"=>[{"thread_id"=>41, "name"=>"[main]<jdbc", "current_call"=>"[...]/vendor/bundle/jruby/2.5.0/gems/rufus-scheduler-3.0.9/lib/rufus/scheduler.rb:170:in `join'"}, {"thread_id"=>33, "name"=>"[main]>worker0", "current_call"=>"[...]/logstash-core/lib/logstash/java_pipeline.rb:279:in `block in start_workers'"}, {"thread_id"=>34, "name"=>"[main]>worker1", "current_call"=>"[...]/logstash-core/lib/logstash/java_pipeline.rb:279:in `block in start_workers'"}, {"thread_id"=>35, "name"=>"[main]>worker2", "current_call"=>"[...]/logstash-core/lib/logstash/java_pipeline.rb:279:in `block in start_workers'"}, {"thread_id"=>36, "name"=>"[main]>worker3", "current_call"=>"[...]/logstash-core/lib/logstash/java_pipeline.rb:279:in `block in start_workers'"}, {"thread_id"=>37, "name"=>"[main]>worker4", "current_call"=>"[...]/logstash-core/lib/logstash/java_pipeline.rb:279:in `block in start_workers'"}, {"thread_id"=>38, "name"=>"[main]>worker5", "current_call"=>"[...]/logstash-core/lib/logstash/java_pipeline.rb:279:in `block in start_workers'"}, {"thread_id"=>39, "name"=>"[main]>worker6", "current_call"=>"[...]/logstash-core/lib/logstash/java_pipeline.rb:279:in `block in start_workers'"}, {"thread_id"=>40, "name"=>"[main]>worker7", "current_call"=>"[...]/logstash-core/lib/logstash/java_pipeline.rb:279:in `block in start_workers'"}]}}
[2020-06-20T01:40:55,397][WARN ][org.logstash.execution.ShutdownWatcherExt] {"inflight_count"=>0, "stalling_threads_info"=>{"other"=>[{"thread_id"=>41, "name"=>"[main]<jdbc", "current_call"=>"[...]/vendor/bundle/jruby/2.5.0/gems/rufus-scheduler-3.0.9/lib/rufus/scheduler.rb:170:in `join'"}, {"thread_id"=>33, "name"=>"[main]>worker0", "current_call"=>"[...]/logstash-core/lib/logstash/java_pipeline.rb:279:in `block in start_workers'"}, {"thread_id"=>34, "name"=>"[main]>worker1", "current_call"=>"[...]/logstash-core/lib/logstash/java_pipeline.rb:279:in `block in start_workers'"}, {"thread_id"=>35, "name"=>"[main]>worker2", "current_call"=>"[...]/logstash-core/lib/logstash/java_pipeline.rb:279:in `block in start_workers'"}, {"thread_id"=>36, "name"=>"[main]>worker3", "current_call"=>"[...]/logstash-core/lib/logstash/java_pipeline.rb:279:in `block in start_workers'"}, {"thread_id"=>37, "name"=>"[main]>worker4", "current_call"=>"[...]/logstash-core/lib/logstash/java_pipeline.rb:279:in `block in start_workers'"}, {"thread_id"=>38, "name"=>"[main]>worker5", "current_call"=>"[...]/logstash-core/lib/logstash/java_pipeline.rb:279:in `block in start_workers'"}, {"thread_id"=>39, "name"=>"[main]>worker6", "current_call"=>"[...]/logstash-core/lib/logstash/java_pipeline.rb:279:in `block in start_workers'"}, {"thread_id"=>40, "name"=>"[main]>worker7", "current_call"=>"[...]/logstash-core/lib/logstash/java_pipeline.rb:279:in `block in start_workers'"}]}}
[2020-06-20T01:40:57,608][FATAL][logstash.runner          ] SIGINT received. Terminating immediately..
[2020-06-20T01:40:57,664][ERROR][org.logstash.Logstash    ] org.jruby.exceptions.ThreadKill
[2020-06-20T20:33:29,580][WARN ][logstash.config.source.multilocal] Ignoring the 'pipelines.yml' file because modules or command line options are specified
[2020-06-20T20:33:29,699][INFO ][logstash.runner          ] Starting Logstash {"logstash.version"=>"7.7.1"}
[2020-06-20T20:33:31,474][INFO ][org.reflections.Reflections] Reflections took 28 ms to scan 1 urls, producing 21 keys and 41 values 
[2020-06-20T20:33:31,824][WARN ][logstash.outputs.elasticsearch] You are using a deprecated config setting "document_type" set in elasticsearch. Deprecated settings will continue to work, but are scheduled for removal from logstash in the future. Document types are being deprecated in Elasticsearch 6.0, and removed entirely in 7.0. You should avoid this feature If you have any questions about this, please visit the #logstash channel on freenode irc. {:name=>"document_type", :plugin=><LogStash::Outputs::ElasticSearch index=>"user_info_index", id=>"2d2a90d5f8aa8b9d46930cded2d613577bb3fc1d5a8ba8b12facb40a6a4290eb", document_id=>"%{id}", hosts=>[http://localhost:9200], document_type=>"user_info", enable_metric=>true, codec=><LogStash::Codecs::Plain id=>"plain_78122c7d-8d0a-4321-a32b-88a098ed5ba2", enable_metric=>true, charset=>"UTF-8">, workers=>1, manage_template=>true, template_name=>"logstash", template_overwrite=>false, doc_as_upsert=>false, script_type=>"inline", script_lang=>"painless", script_var_name=>"event", scripted_upsert=>false, retry_initial_interval=>2, retry_max_interval=>64, retry_on_conflict=>1, ilm_enabled=>"auto", ilm_rollover_alias=>"logstash", ilm_pattern=>"{now/d}-000001", ilm_policy=>"logstash-policy", action=>"index", ssl_certificate_verification=>true, sniffing=>false, sniffing_delay=>5, timeout=>60, pool_max=>1000, pool_max_per_route=>100, resurrect_delay=>5, validate_after_inactivity=>10000, http_compression=>false>}
[2020-06-20T20:33:33,658][INFO ][logstash.outputs.elasticsearch][main] Elasticsearch pool URLs updated {:changes=>{:removed=>[], :added=>[http://localhost:9200/]}}
[2020-06-20T20:33:33,825][WARN ][logstash.outputs.elasticsearch][main] Restored connection to ES instance {:url=>"http://localhost:9200/"}
[2020-06-20T20:33:33,880][INFO ][logstash.outputs.elasticsearch][main] ES Output version determined {:es_version=>6}
[2020-06-20T20:33:33,885][WARN ][logstash.outputs.elasticsearch][main] Detected a 6.x and above cluster: the `type` event field won't be used to determine the document _type {:es_version=>6}
[2020-06-20T20:33:33,908][WARN ][logstash.outputs.elasticsearch][main] DEPRECATION WARNING: Connecting to an OSS distribution of Elasticsearch using the default distribution of Logstash will stop working in Logstash 8.0.0. Please upgrade to the default distribution of Elasticsearch, or use the OSS distribution of Logstash {:url=>"http://localhost:9200/"}
[2020-06-20T20:33:33,927][INFO ][logstash.outputs.elasticsearch][main] New Elasticsearch output {:class=>"LogStash::Outputs::ElasticSearch", :hosts=>["http://localhost:9200"]}
[2020-06-20T20:33:33,984][INFO ][logstash.outputs.elasticsearch][main] Using default mapping template
[2020-06-20T20:33:34,018][INFO ][logstash.outputs.elasticsearch][main] Index Lifecycle Management is set to 'auto', but will be disabled - Your Elasticsearch cluster is before 7.0.0, which is the minimum version required to automatically run Index Lifecycle Management
[2020-06-20T20:33:34,021][INFO ][logstash.outputs.elasticsearch][main] Attempting to install template {:manage_template=>{"template"=>"logstash-*", "version"=>60001, "settings"=>{"index.refresh_interval"=>"5s"}, "mappings"=>{"_default_"=>{"dynamic_templates"=>[{"message_field"=>{"path_match"=>"message", "match_mapping_type"=>"string", "mapping"=>{"type"=>"text", "norms"=>false}}}, {"string_fields"=>{"match"=>"*", "match_mapping_type"=>"string", "mapping"=>{"type"=>"text", "norms"=>false, "fields"=>{"keyword"=>{"type"=>"keyword", "ignore_above"=>256}}}}}], "properties"=>{"@timestamp"=>{"type"=>"date"}, "@version"=>{"type"=>"keyword"}, "geoip"=>{"dynamic"=>true, "properties"=>{"ip"=>{"type"=>"ip"}, "location"=>{"type"=>"geo_point"}, "latitude"=>{"type"=>"half_float"}, "longitude"=>{"type"=>"half_float"}}}}}}}}
[2020-06-20T20:33:34,056][WARN ][org.logstash.instrument.metrics.gauge.LazyDelegatingGauge][main] A gauge metric of an unknown type (org.jruby.specialized.RubyArrayOneObject) has been created for key: cluster_uuids. This may result in invalid serialization.  It is recommended to log an issue to the responsible developer/development team.
[2020-06-20T20:33:34,064][INFO ][logstash.javapipeline    ][main] Starting pipeline {:pipeline_id=>"main", "pipeline.workers"=>8, "pipeline.batch.size"=>125, "pipeline.batch.delay"=>50, "pipeline.max_inflight"=>1000, "pipeline.sources"=>["C:/Users/bahma/eclipse-workspace/project/logstash/bin/logstash-mysql.conf"], :thread=>"#<Thread:0x65a3e7df run>"}
[2020-06-20T20:33:35,417][INFO ][logstash.javapipeline    ][main] Pipeline started {"pipeline.id"=>"main"}
[2020-06-20T20:33:35,474][INFO ][logstash.agent           ] Pipelines running {:count=>1, :running_pipelines=>[:main], :non_running_pipelines=>[]}
[2020-06-20T20:33:35,872][INFO ][logstash.agent           ] Successfully started Logstash API endpoint {:port=>9600}
[2020-06-20T20:34:01,266][INFO ][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] (0.085210s) SELECT user_id as id,languages,middlewares,databasesList,operating_system,experience,frameworks,i.cus_name,i.cus_addr,i.cus_dob,i.cus_email,i.cus_tel FROM user_cv INNER JOIN user_info i ON user_cv.user_id = i.CUS_USER_ID WHERE user_cv.user_id > 0
[2020-06-20T20:35:00,367][INFO ][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] (0.001494s) SELECT user_id as id,languages,middlewares,databasesList,operating_system,experience,frameworks,i.cus_name,i.cus_addr,i.cus_dob,i.cus_email,i.cus_tel FROM user_cv INNER JOIN user_info i ON user_cv.user_id = i.CUS_USER_ID WHERE user_cv.user_id > 7502
[2020-06-20T20:36:00,273][INFO ][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] (0.000842s) SELECT user_id as id,languages,middlewares,databasesList,operating_system,experience,frameworks,i.cus_name,i.cus_addr,i.cus_dob,i.cus_email,i.cus_tel FROM user_cv INNER JOIN user_info i ON user_cv.user_id = i.CUS_USER_ID WHERE user_cv.user_id > 7502
[2020-06-20T20:37:00,189][INFO ][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] (0.001308s) SELECT user_id as id,languages,middlewares,databasesList,operating_system,experience,frameworks,i.cus_name,i.cus_addr,i.cus_dob,i.cus_email,i.cus_tel FROM user_cv INNER JOIN user_info i ON user_cv.user_id = i.CUS_USER_ID WHERE user_cv.user_id > 7502
[2020-06-20T20:38:00,113][INFO ][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] (0.000918s) SELECT user_id as id,languages,middlewares,databasesList,operating_system,experience,frameworks,i.cus_name,i.cus_addr,i.cus_dob,i.cus_email,i.cus_tel FROM user_cv INNER JOIN user_info i ON user_cv.user_id = i.CUS_USER_ID WHERE user_cv.user_id > 7502
[2020-06-20T20:39:00,332][INFO ][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] (0.001127s) SELECT user_id as id,languages,middlewares,databasesList,operating_system,experience,frameworks,i.cus_name,i.cus_addr,i.cus_dob,i.cus_email,i.cus_tel FROM user_cv INNER JOIN user_info i ON user_cv.user_id = i.CUS_USER_ID WHERE user_cv.user_id > 7502
[2020-06-20T20:40:00,252][INFO ][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] (0.000936s) SELECT user_id as id,languages,middlewares,databasesList,operating_system,experience,frameworks,i.cus_name,i.cus_addr,i.cus_dob,i.cus_email,i.cus_tel FROM user_cv INNER JOIN user_info i ON user_cv.user_id = i.CUS_USER_ID WHERE user_cv.user_id > 7502
[2020-06-20T20:41:00,151][INFO ][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] (0.000827s) SELECT user_id as id,languages,middlewares,databasesList,operating_system,experience,frameworks,i.cus_name,i.cus_addr,i.cus_dob,i.cus_email,i.cus_tel FROM user_cv INNER JOIN user_info i ON user_cv.user_id = i.CUS_USER_ID WHERE user_cv.user_id > 7502
[2020-06-20T20:42:00,072][INFO ][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] (0.000971s) SELECT user_id as id,languages,middlewares,databasesList,operating_system,experience,frameworks,i.cus_name,i.cus_addr,i.cus_dob,i.cus_email,i.cus_tel FROM user_cv INNER JOIN user_info i ON user_cv.user_id = i.CUS_USER_ID WHERE user_cv.user_id > 7502
[2020-06-20T20:43:00,295][INFO ][logstash.inputs.jdbc     ][main][33c61d05e2b54c4d667618d2fc05094aae73d2886ad80c5c4c1c22e1e4ee0e2c] (0.000812s) SELECT user_id as id,languages,middlewares,databasesList,operating_system,experience,frameworks,i.cus_name,i.cus_addr,i.cus_dob,i.cus_email,i.cus_tel FROM user_cv INNER JOIN user_info i ON user_cv.user_id = i.CUS_USER_ID WHERE user_cv.user_id > 7502
[2020-06-20T20:43:03,906][WARN ][logstash.runner          ] SIGINT received. Shutting down.
[2020-06-20T20:43:04,320][INFO ][logstash.javapipeline    ] Pipeline terminated {"pipeline.id"=>"main"}
[2020-06-20T20:43:04,722][INFO ][logstash.runner          ] Logstash shut down.
